app.js

```diff
@@ -51,4 +51,4 @@ form.addEventListener('submit', (event) => {
   displayBooks();
 });
 
-displayBooks();
+// displayBooks();
```

Comment:
- [ ] Please remove `app.js` file because now you're using another script file `class.js` to make your repo clean from any unused code.